core/sdk/src/api/firebolt/fb_metrics.rs

```diff
@@ -537,7 +741,8 @@ impl ExtnPayloadProvider for MetricsResponse {
 #[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
 pub enum MetricsPayload {
     BehaviorMetric(BehavioralMetricPayload, CallContext),
-    OperationalMetric(TelemetryPayload),
+    TelemetryPayload(TelemetryPayload),
+    OperationalMetric(OperationalMetricPayload),
```

Comment:
@bobra200 : From POC, I'm not clear on if we're keeping both, and if so, why?

Reply:
Telemetry = triage (legacy)
OperationalMetric = low cardinality, closer to "realtime"